src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarm.java

```diff
@@ -487,8 +490,45 @@ public ListSuitesResult listSuites(String runArn) {
 
         return api.listSuites(request);
     }
-
-
+    
+    /*
+     * Check if account has unmetered devices based on upload type
+     */
+    public void checkAccountUnmeteredStatus(String appArtifact) throws AWSDeviceFarmException {
+    	int unmeteredDevices = 0; 
+    	String platform;
+    	if (appArtifact.toLowerCase().endsWith("apk")) {
+    		unmeteredDevices = getUnmeteredDevices(appArtifact);
```

Comment:
Inconsistent indentation.

It appears the rest of the project uses spaces and it appears these are tabs. Please unify with whatever was there previously.